client/src/components/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,38 @@
+import { Center, Spinner as PageSpinner } from '@chakra-ui/react'
+
+interface SpinnerProps {
```

Comment:
actually ChakraUI exports all component props, which you can see from the [source](https://github.com/chakra-ui/chakra-ui/blob/main/packages/spinner/src/spinner.tsx), so you can just import `SpinnerProps`